tests/lib/prepare-restore.sh

```diff
@@ -494,19 +494,26 @@ prepare_project() {
     # base on the packaging. In Fedora/Suse this is handled via mock/osc
     case "$SPREAD_SYSTEM" in
         debian-*|ubuntu-*)
-            # in 16.04: apt build-dep -y ./
+            best_golang=golang-1.13
             if [[ "$SPREAD_SYSTEM" == debian-9-* ]]; then
-                best_golang="$(python3 ./tests/lib/best_golang.py)"
-                test -n "$best_golang"
-                sed -i -e "s/golang-1.10/$best_golang/" ./debian/control
-            else
-                best_golang=golang-1.10
+                echo "debian-9 tests disabled (no golang-1.13)"
+                exit 1
```

Comment:
I assume this means we need to drop `debian-9-*` from the systems in spread.yaml that we support, correct?

Also I assume this means we no longer support debian 9 since we can't build new snapd's on it and we won't be testing it, is that accurate?

Reply:
That is a really good question. AFAICT we stopped running our CI against debian-9 a while ago. I would love to keep supporting debian-9, maybe even get it back into CI. They support until June 2022